recipe/meta.yaml

```diff
@@ -9,34 +9,34 @@ source:
   sha256: ce6abd3f1d39d42333b07c608ae022ac09a3f5984b297b21436c3389d7b77af5
 
 build:
-  number: 1
+  number: 2
   # This is a noarch: python package, it contains no compiled code but a
   # compiler is required to convert protocol buffer definitions.
   noarch: python
   ignore_run_exports:
-    - libgcc-ng
-    - libstdcxx-ng
+    - {{ stdlib("c") }}
+    - {{ compiler("cxx") }}
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - {{ compiler("c") }}
```

Comment:
Is this package noarch? It does depend on a compiler, right?

Reply:
Sure, but I don't think it will work with CFEP-25 if it needs a compiler.

Reply:
That's the reason the below exists?

```
  ignore_run_exports:
    - {{ stdlib("c") }}
    - {{ compiler("cxx") }}
```

Reply:
@ocefpaf Changed the title. So noarch but not CFEP-25.